docs/ad/index.md

```diff
@@ -55,12 +55,39 @@ For example, if you choose `min()`, the detector focuses on finding anomalies ba
 You can add a maximum of five features for a detector.
 {: .note }
 
-1. On the **Features** page, select **Add features**.
-1. Enter the **Name** of the feature.
+1. On the **Model configuration** page, enter the **Feature name**.
 1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the **field** and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
-1. Preview sample anomalies and adjust the feature settings if needed.
-- For sample previews, the anomaly detection plugin selects a small number of data samples---for example, one data point every 30 minutes---and uses interpolation to estimate the remaining data points to approximate the actual feature data. It loads this sample dataset into the detector. The detector uses this sample dataset to generate a sample preview of anomaly results.
+
+#### (Optional) Set a category field
```

Comment:
can we call out that this is for the support of high cardinality?

Reply:
added